core/AutocorrectSuggestion.h

```diff
@@ -6,10 +6,11 @@
 namespace sorbet::core {
 
 struct AutocorrectSuggestion {
-    core::Loc loc;
-    std::string replacement;
+    std::string message;
+    std::vector<std::pair<core::Loc, std::string>> edits;
```

Comment:
We're trying not to use `pair` in public api, but use structs with descriptive names instead

Reply:
ack, I'll make a `struct Edit` instead.